zimui/src/components/ErrorDisplay.vue

```diff
@@ -8,25 +8,82 @@ const errimage = ref(errimageData)
 </script>
 
 <template>
-  <div class="error-container">
-    <img :src="errimage" class="error-image" />
-    <p class="error-text">{{ main.errorMessage }}</p>
-    <p class="error-text">{{ main.errorDetails }}</p>
+  <div class="error-page">
+    <div class="error-container">
+      <div class="error-content">
+        <p class="error-text">{{ main.errorMessage }}</p>
+        <p class="error-text">{{ main.errorDetails }}</p>
+      </div>
+      <div class="error-image-container">
+        <img :src="errimage" class="error-image" alt="error image" />
+      </div>
+    </div>
   </div>
 </template>
 
 <style scoped>
-.error-container {
+body {
+  margin: 0;
+  background-color: #f0f4f8;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Comment:
Sorry, one last point, can you please explain why you change font for this error page?

Reply:
Because i wanted to make it better. should i remove it

Reply:
Why only on error page then? And how is it better? Keep in mind that we need to rely on fonts which are available offline.

Reply:
But how is this better? In the screenshot, it looks like the fonts are very "coarse" with significant "aliasing artefact", it feels weird.

Reply:
so which one do you suggest.

Reply:
I suggest that we continue to use the normal font which is used everywhere in the UI unless you have arguments against this.

The whole UI font is setup in https://github.com/openzim/kolibri/blob/21085d4e5ea660b2048ef4cff8cab91157238d1e/zimui/src/App.vue#L9 ; this setup ensures (or should ensure) that we use Lato everywhere, with antialiasing turned on when possible. And Lato is embedded inside the ZIM. I don't recall exactly why we chose Lato in the first place, but I don't get (yet) why this error should have a different font.

Reply:
the changes are made as you suggested.